cmd/fyne/internal/commands/release.go

```diff
@@ -176,6 +183,9 @@ func (r *Releaser) Run(params []string) {
 		return
 	}
 
+	if r.verbose {
+		mobile.Verbose = true
+	}
```

Comment:
Should it not be enough to do `mobile.Verbose = r.verbose`?

Reply:
done